lighthouse-core/lib/tracehouse/trace-processor.js

```diff
@@ -513,7 +522,7 @@ class TraceProcessor {
    *
    * @param {LH.TraceEvent[]} events
    * @param {LH.TraceEvent} timeOriginEvent
-   * @return {{lcp: LCPEvent | undefined, invalidated: boolean}}
+   * @return {{lcp: LCPEvent | undefined}}
```

Comment:
it's unclear if this (and `fmpFellBack`) is a necessary change for some purpose or if it's just a clearing out of stuff no one actually uses